Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,46 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow.git", :tag => "v1.0.0",
+    :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    ENV["PYTHON_BIN_PATH"]=`which python`.strip
+    ENV["CC_OPT_FLAGS"]="-march=native"
+    ENV["TF_NEED_JEMALLOC"]="1"
+    ENV["TF_NEED_GCP"]="0"
+    ENV["TF_NEED_HDFS"]="0"
+    ENV["TF_ENABLE_XLA"]="0"
+    ENV["USE_DEFAULT_PYTHON_LIB_PATH"]="1"
+    ENV["TF_NEED_OPENCL"]="0"
+    ENV["TF_NEED_CUDA"]="0"
+    system "./configure"
+
+    system "bazel", "build", "--compilation_mode=opt", "--copt=-march=native", "tensorflow:libtensorflow_c.so"
```

Comment:
Oh, my comment was about the bazel targets and not the filename.
I was suggesting that the build command use the bazel target [`//tensorflow:libtensorflow.so`](https://github.com/tensorflow/tensorflow/blob/e75c0fc/tensorflow/BUILD#L300) instead of [`//tensorflow:libtensorflow_c.so`](https://github.com/tensorflow/tensorflow/blob/e75c0fc/tensorflow/BUILD#L309) (which is what is being done right now)